plugins/fontpreview.lua

```diff
@@ -0,0 +1,63 @@
+--mod-version:3 --priority:110
+
+local common = require 'core.common'
+local style = require 'core.style'
+local View = require 'core.view'
+
+local FontView = View:extend()
+
+local font_text = "The quick brown fox jumps over the lazy dog."
+
+function FontView:new(path)
+	FontView.super.new(self)
+	self.path = path
+	self.fonts = {}
+	for i=1, 8 do self.fonts[i] = renderer.font.load(path, 12+i*7) end
```

Comment:
Might be a good idea to scale the font size by `SCALE`.

Reply:
remind me why does `SCALE` exist in the first place lol

Reply:
Because your monitor could be scaled.
Or you just want the UI scaled up or down.